src/components/AppBar/UserMenu.vue

```diff
@@ -60,19 +58,18 @@ export default {
             );
           }
         }
-        // If first name + last name aren't specified, try to use the login instead
-        const { login } = this.user;
-        if (login) {
-          return login.slice(0, 2);
-        }
```

Comment:
Is there a reason you're axing this? I'm not morally opposed to just having `??` for users with no name defined in GH, but I just want to make sure the decision is deliberate.

Reply:
got me there, I thought we were still providing that. Exterminate